api/app.py

```
@@ -13,12 +15,30 @@
 from logging_config import setup_logging
-import config
 
 setup_logging()
 logger = logging.getLogger(__name__)
 
+
+@asynccontextmanager
+async def lifespan(app: FastAPI):
+    """Управление жизненным циклом приложения"""
+    # Startup
+    asyncio.create_task(check_for_new_rss_items())
```

Ruff (0.14.4)

Store a reference to the return value of `asyncio.create_task`

(RUF006)